etc/run-auth-ldap.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# see run-jupyterhub.sh for setting up local mail delivery
+export SIREPO_FROM_EMAIL='support@radiasoft.net'
+export SIREPO_FROM_NAME='RadiaSoft Support'
+export SIREPO_SMTP_PASSWORD='vagrant'
+# POSIT: same as sirepo.smtp.DEV_SMTP_SERVER
+export SIREPO_SMTP_SERVER='dev'
+export SIREPO_SMTP_USER='vagrant'
+export SIREPO_AUTH_METHODS='email:guest:ldap'
+export PYKERN_PKDEBUG_WANT_PID_TIME=1
+
+# primary cause of failed wheel generation
+pip install --upgrade pip
+echo "upgraded pip"
+
+# install prerequisites
+sudo yum groupinstall "Development tools"
+sudo yum install openldap-devel python-devel
+echo "installed prerequisites"
+
+# secondary cause of failed wheel generation
+pip install --upgrade setuptools wheel
+echo "installed wheel setup"
+
+# main installs
+pip install python-ldap
```

Comment:
Do not install this.

Reply:
Meaning install this in normal dev instead of in run-auth? The login API is written with python-ldap.

Reply:
python-ldap installs the libraries in /usr where the "system" (red hat) python is installed. The python we use is in `~/.pyenv` which doesn't look in /usr for python libraries. What you want to do is probably "pip install ldap"

Reply:
I believe python-ldap is the correct package. 

`pip install ldap` won't work. See the docs https://pypi.org/project/ldap/ .

For me it seems `pip install python-ldap` installed in the right place
```
~$ find ~/.pyenv/ -name 'ldap'
~$ pip install python-ldap
<snip>
Successfully installed python-ldap-3.4.3
~$ find ~/.pyenv/ -name 'ldap'
/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/ldap
```

Reply:
I'm sorry, I was confused. I thought "python-ldap" was a yum install. pip install is correct. ignore my comment.